lib/mutant/integration/minitest.rb

```diff
@@ -0,0 +1,155 @@
+require 'minitest'
+
+# monkey patch for minitest
+module Minitest
+  # Prevent autorun from running tests when the VM closes
+  #
+  # Mutant needs control about the exit status of the VM and
+  # the moment of test execution
+  #
+  # @api private
+  #
+  # @return [nil]
+  def self.autorun
+  end
+
+end # Minitest
```

Comment:
Reopening the scope `Minitest` is not needed technically, but maybe for consistency?